static/app/views/dashboardsV2/widgetBuilder/buildSteps/filterResultsStep/issuesSearchBar.tsx

```diff
@@ -19,29 +19,27 @@ interface Props {
   onBlur: SearchBarProps['onBlur'];
   onSearch: SearchBarProps['onSearch'];
   organization: Organization;
-  query: WidgetQuery;
-  selection: PageFilters;
+  pageFilters: PageFilters;
   tags: TagCollection;
-  searchSource?: string;
+  widgetQuery: WidgetQuery;
 }
 
 function IssuesSearchBarContainer({
   tags,
   onSearch,
   onBlur,
+  widgetQuery,
   organization,
-  query,
-  selection,
-  searchSource,
+  pageFilters,
 }: Props) {
   const api = useApi();
   function tagValueLoader(key: string, search: string) {
-    const orgId = organization.slug;
-    const projectIds = selection.projects.map(id => id.toString());
+    const orgId = organization!.slug;
+    const projectIds = pageFilters?.projects.map(id => id.toString());
```

Comment:
`projects` and `datetime` aren't optional in `pageFilters` according to the type, can we drop `?`?